### DISCUSSION TITLE
Access 'create_model" atttibutes

### DISCUSSION THREAD

**Discussion by usyed15181** [pycaret/pycaret#1524] [Q&A] [answered]
Hi, 

I am using 'pycaret.clustering' and want to save model attribute values like 'Silhouette', 'Calinski-Harabasz' in a dataframe.

How can i access these values?

I tried:
model = create_model('kmeans')
print(model.Silhouette)

AttributeError: 'KMeans' object has no attribute 'Silhouette'

**Comment by Yard1** [comment, marked as answer]
Do `pull()` right after `create_model`.

**Reply by usyed15181** [reply]
yup, worked! Thanks.
